bundler/spec/dependabot/bundler/file_parser_spec.rb

```diff
@@ -153,17 +136,7 @@
     end
 
     context "from a gems.rb and gems.locked" do
-      let(:gemfile) do
-        Dependabot::DependencyFile.new(name: "gems.rb", content: gemfile_body)
-      end
-      let(:lockfile) do
-        Dependabot::DependencyFile.new(
-          name: "gems.locked",
-          content: lockfile_body
-        )
-      end
-      let(:gemfile_fixture_name) { "version_specified" }
-      let(:lockfile_fixture_name) { "bundler_2.lock" }
+      let(:dependency_files) { project_dependency_files("bundler1/version_specified_bundler_2") }
```

Comment:
Not for now but adding a mental note to go through fixtures once we add support for bundler 2 and move to `bundler2/..`